app/models/user.rb

```diff
@@ -2,7 +2,12 @@ class User < ApplicationRecord
   SIGN_UP_REQUIRE_ATTRIBUTES = %i(name email password
 password_confirmation).freeze
 
-  attr_accessor :remember_token
+  attr_accessor :remember_token, :activation_token
+
+  before_save :downcase_email
+  before_create :create_activation_digest
+
+  scope :activated, ->{where "activated = true"}
```

Comment:
```suggestion
  scope :activated, ->{where activated: true}
```
thử thế này xem có chạy ko em

Reply:
dạ em vừa push lên lại á anh, trùng hợp em vừa test xong cái anh comment luôn :)))